packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -10,185 +10,150 @@ import { setup } from '../setupBasicMints';
-        units: assays[1].makeUnits(7),
-      },
-    ]);
+    const option = aliceSeat.makeCallOption();
 
-    // 3: Imagine that Alice sends the invite to Bob (not done here
+    // Imagine that Alice sends the option to Bob for free (not done here
     // since this test doesn't actually have separate vats/parties)
 
-    // 4: Bob inspects the invite payment and checks that it is the
+    // 4: Bob inspects the option (an invite payment) and checks that it is the
     // contract instance that he expects as well as that Alice has
     // already escrowed.
 
-    const bobIntendedOfferRules = harden({
+    const inviteAssay = zoe.getInviteAssay();
+    const bobExclOption = await inviteAssay.claimAll(option);
+    const optionExtent = bobExclOption.getBalance().extent;
+    const { installationHandle } = zoe.getInstance(optionExtent.instanceHandle);
+    t.equal(installationHandle, coveredCallInstallationHandle);
+    t.equal(optionExtent.seatDesc, 'exerciseOption');
+    t.ok(unitOps[0].equals(optionExtent.underlyingAsset, moola(3)));
+    t.ok(unitOps[1].equals(optionExtent.strikePrice, simoleans(7)));
+    t.equal(optionExtent.expirationDate, 1);
+    t.deepEqual(optionExtent.timerAuthority, timer);
+
+    const bobPayments = [undefined, bobSimoleanPayment];
+
+    const bobOfferRules = harden({
       payoutRules: [
         {
           kind: 'wantAtLeast',
-          units: assays[0].makeUnits(3),
+          units: optionExtent.underlyingAsset,
         },
         {
           kind: 'offerAtMost',
-          units: assays[1].makeUnits(7),
+          units: optionExtent.strikePrice,
         },
       ],
       exitRule: {
         kind: 'onDemand',
       },
     });
 
-    const inviteExtent = bobInvitePayment.getBalance().extent;
-    t.equal(inviteExtent.instanceHandle, instanceHandle);
-    t.ok(sameStructure(inviteExtent.offerMadeRules, aliceOfferRules));
-    t.ok(
-      sameStructure(
-        inviteExtent.offerToBeMade,
-        bobIntendedOfferRules.payoutRules,
-      ),
-    );
-
-    // Bob claims all with the Zoe inviteAssay
-    const inviteAssay = zoe.getInviteAssay();
-    const bobExclInvitePayment = await inviteAssay.claimAll(bobInvitePayment);
-
-    // 5: Only after assaying the invite does he unwrap it (destroying
-    // the ERTP invite) and accept it
-    const bobInvite = await bobExclInvitePayment.unwrap();
-    const bobPayments = [undefined, bobSimoleanPayment];
-
     // 6: Bob escrows
-    const {
-      escrowReceipt: allegedBobEscrowReceipt,
-      payout: bobPayoutP,
-    } = await zoe.escrow(bobIntendedOfferRules, bobPayments);
-
-    // 7: Bob does a claimAll on the escrowReceipt payment
-    const bobEscrowReceipt = await escrowReceiptAssay.claimAll(
-      allegedBobEscrowReceipt,
+    const { seat: bobSeat, payout: bobPayoutP } = await zoe.redeem(
+      bobExclOption,
+      bobOfferRules,
+      bobPayments,
     );
 
     // 8: Bob makes an offer with his escrow receipt
```

Comment:
What does it mean that this testcase has steps 6,8,13,14 but no steps 7,9,10,11,12 ?

Reply:
It means that trying to number the steps of a test when the test changes drastically from week to week was a bad idea. :) I'll remove the numbers.

Reply:
Ok, those numbered steps should be removed now